avocado/utils/nvme.py

```diff
@@ -55,7 +55,7 @@ def get_number_of_ns_supported(controller_name):
     :rtype: integer
     """
     cmd = f"nvme id-ctrl /dev/{controller_name}"
-    out = process.run(cmd, ignore_status=True, sudo=True, shell=True).stdout_text
+    out = process.run(cmd, ignore_status=True, shell=True).stdout_text
```

Comment:
why is this change required? are we not suppose to run this command having root

Reply:
> why is this change required? are we not suppose to run this command having root

Done, changed it back